src/internal/_clone.js

```diff
@@ -9,9 +9,10 @@ var type = require('../type');
  * @param {*} value The value to be copied
  * @param {Array} refFrom Array containing the source references
  * @param {Array} refTo Array containing the copied source references
+ * @param {Boolean} deep Whether or not to perform deep cloning.
  * @return {*} The copied value.
```

Comment:
The implementation is not so trivial so I'm happy there is some documentation there.